scripts/fix_any.py

```diff
@@ -0,0 +1,130 @@
+"""Simple script to fix Any's in bad files created by datamodel-code-generator."""
+
+# This relies on datamodel-codegen 0.5.28 to create classes that work,
+# but don't do much checking.
+# This is because the generated classes rely on Any - which does no checking.
+# This script changes the Any's appropriately from the plural form of the needed
+# class to the singular form.
+# It then reorders the classes so there are no forwards required.
+# This reordering is optimistic and assumes there is nothing circular involved -
+# and that there are no side effect classes that become forwarded.
+
+import os
+import sys
+
+pattern1 = 'ies: Optional[Dict[str, Any]]'
+pattern2 = 's: Optional[Dict[str, Any]]'
```

Comment:
Should we have a catch pattern which is just Dict[str, Any] to test for when we can potentially escape it.